.github/workflows/yarn-upgrade.yml

```diff
@@ -33,9 +33,13 @@ jobs:
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |-
             ${{ runner.os }}-yarn-
+      - name: Yarn Install
+        run: yarn install --frozen-lockfile
       - name: Install Tools
         run: |-
           npm -g install lerna npm-check-updates@^9.0.0
+      - name: Build CLI
+        run: cd packages/aws-cdk && ../../scripts/buildup
```

Comment:
We need to build the CLI in order to generate attributions (the bundler looks for the `.js` files)